util/text/text.go

```diff
@@ -1,12 +1,18 @@
 package text
 
 import (
-	"bytes"
-	"strings"
-
 	"github.com/mattn/go-runewidth"
+	"strings"
+	"unicode/utf8"
 )
 
+// Force runewidth not to treat ambiguous runes as wide chars, so that things
+// like unicode ellipsis/up/down/left/right glyphs can have correct runewidth
+// and can be displayed correctly in terminals.
+func init() {
```

Comment:
`util/text/left_padded.go` and `util/text/text.go` are in the same package (`text`) so a single `init()` will be enough.

Reply:
Thanks for pointing this out and helpping me understand it. I am fixing it.